apps/web/src/actions/workspaces/update.ts

```diff
@@ -21,9 +22,9 @@ export const updateWorkspaceAction = authProcedure
       .find(input.workspaceId)
       .then((r) => r.unwrap())
 
-    const updatedWorkspace = await updateWorkspace({
-      workspace,
+    const updatedWorkspace = await updateWorkspace(workspace, {
       name: input.name,
+      defaultProviderId: input.defaultProviderId,
```

Comment:
These kind of changes would be better use a different action. Not generic workspace update. This way we can track an event and see how many users set default provider

Reply:
Not necessary to change unless we think is interesting to track this as event. Ask @cesr

Reply:
Okay, will ask, although we could do something like `if (defaultProviderId) trackEvent()` in the action itself.

Reply:
Yes, but it is better to have actions that do one thing: `create workspace`, `update workspace`, `add default provider`.

Yes, adding a default provider is a workspace update. But is more concrete. So let's make concrete from the action.